src/platforms/common/index.mdx

```diff
@@ -0,0 +1,39 @@
+On this page, we get you up and running with Sentry's SDK, so that it will automatically report errors and exceptions in your application.
+
+<Note>
+<markdown>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</markdown>
+</Note>
+
+<PlatformContent includePath="framework-list" />
+
+## Install
+
+Sentry captures data by using an SDK within your application’s runtime:
+
+<PlatformContent includePath="getting-started-install" />
+
+## Initialize
+
+`init` Sentry as soon as possible in your app. Once this is done, the Sentry SDK captures all unhandled exceptions.
+
+<PlatformContent includePath="getting-started-config" />
+
+<Alert level="note" title="Note"><markdown>
```

Comment:
Make it a ``<Note>`` (or delete it) given it literally doesnt matter to the developer right here, and they almost certainly already have been given it before.

Reply:
Moved this to the proper Note format. DSN is one of our most-searched terms, so I think worth keeping it here.